app/src/org/commcare/activities/HomeScreenBaseActivity.java

```diff
@@ -358,7 +358,7 @@ protected void onSaveInstanceState(Bundle outState) {
     }
 
     @Override
-    protected void onActivityResult(int requestCode, int resultCode, Intent intent) {
+    protected void onActivityResultSessionSafe(int requestCode, int resultCode, Intent intent) {
```

Comment:
Should fix https://fabric.io/dimagi/android/apps/org.commcare.dalvik/issues/59c3fc75be077a4dcc02c8f0?time=last-thirty-days